src/OpenIddict.Abstractions/Resources/xlf/OpenIddictResources.nl.xlf

```diff
@@ -0,0 +1,483 @@
+﻿<?xml version="1.0" encoding="UTF-8"?>
+<xliff version="1.2" xmlns="urn:oasis:names:tc:xliff:document:1.2">
+  <file datatype="xml" source-language="en" target-language="nl" original="../OpenIddictResources.resx">
+    <body>
+      <trans-unit id="ID2000" datatype="html">
+        <source>The security token is missing.</source>
+        <target>Het beveiligingstoken ontbreekt.</target>
```

Comment:
Looks like none of the `target` nodes has a `state="translated"` attribute, which may not make Arcade happy. What tool did you use?

Reply:
POEditor, will search/replace in the next commit.